tests/native/test_unicorn_concrete.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import os
+
+from manticore.native import Manticore
+from manticore.core.plugin import Plugin
+
+
+class ConcretePlugin(Plugin):
+
+    def will_start_run_callback(self, state, *_args):
+        state.cpu.emulate_until(0)
+
+
+class RegisterCapturePlugin(Plugin):
+
+    def will_start_run_callback(self, state, *_args):
+        self._cpu = state.cpu
+
+    def did_finish_run_callback(self):
+        with self.manticore.locked_context() as context:
+            for reg in self._cpu.all_registers:
+                context[reg] = getattr(self._cpu, reg)
+
+
+class ManticornTest(unittest.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        dirname = os.path.dirname(__file__)
+        self.m = Manticore(os.path.join(dirname, 'binaries', 'arguments_linux_amd64'),
+                           argv=['argv', 'mc', 'argface'])
+        self.concrete_instance = Manticore(os.path.join(dirname, 'binaries', 'arguments_linux_amd64'),
+                                           argv=['argv', 'mc', 'argface'])
+
+        self.concrete_instance.register_plugin(ConcretePlugin())
+        self.concrete_instance.register_plugin(RegisterCapturePlugin())
+        self.m.register_plugin(RegisterCapturePlugin())
+
+    def test_register_comparison(self):
```

Comment:
Aren't those tests taking a bit long time to execute? If so, it might be good to make both assertions in one test.